### PR TITLE
Feature: Access conversations with deactivated users

### LINKED ISSUE #7425: Help Wanted: Access conversations with deactivated users
# Context

The Mattermost Product Documentation doesn't specify whether users can access conversations with deactivated users.

# Documentation Request

Clarify in the end user [product docs](https://docs.mattermost.com/collaborate/channel-types.html) that any direct or group message history you had with a deactivated user remains available unless your system admin [disables your ability to do so](https://docs.mattermost.com/configure/site-configuration-settings.html#users-viewarchivedchannels).

# QA Validation Criteria

In a generated docs preview:
- Visiting the docs page directly returns a note for both direct and group messages regarding the user's ability to access deactivated users (when enabled).

### PR DESCRIPTION
**Pull request by TomerPacific**
#### Summary
This pull request includes changes to the `source/collaborate/channel-types.rst` file to improve the clarity and structure of the documentation for direct and group messages. The key changes involve replacing `.. tip::` with `.. note::` and adding information about message history with deactivated users.

#### Ticket Link
Resolves #7425

### CODE REVIEW

File: source/collaborate/channel-types.rst [hunk 1]
```diff
@@ -29,9 +29,11 @@ Direct messages conversations between 2 people. You can start a direct message w
...
 
-.. tip::
+.. note::
 
-  From Mattermost v10, when sending a direct message, Mattermost warns you that the recipient's availability is set to :ref:`Do Not Disturb <preferences/set-your-status-availability:set your availability>`, and when the recipient's local time is outside of regular business hours (between 10PM and 6AM). This warning displays directly above the message text field.
+  - From Mattermost v10, when sending a direct message, Mattermost warns you that the recipient's availability is set to :ref:`Do Not Disturb <preferences/set-your-status-availability:set your availability>`, and when the recipient's local time is outside of regular business hours (between 10PM and 6AM). This warning displays directly above the message text field.
+  
+  - Any message history you have with a deactived user remains available :ref:`unless your systems administrator disables your ability to do so <configure/site-configuration-settings:allow users to view archived channels>`.
```
Comment (cwarnermm) [inline source/collaborate/channel-types.rst:36]:
```suggestion
  - Any message history you have with a deactivated user remains available :ref:`unless your system admin disables your ability to do so <configure/site-configuration-settings:allow users to view archived channels>`.
```

File: source/collaborate/channel-types.rst [hunk 2]
```diff
@@ -40,6 +42,8 @@ Group messages are conversations between 3 to 7 people. You can also start a gro
 
 From Mattermost v9.1, group messages increment the numbered badge and trigger a notification unless the direct message is muted, or your notifications are disabled. You can control how you're notified about group message conversations by going to **Settings > Notifications**. See the :doc:`notification documentation </preferences/manage-your-notifications>` to learn more.
 
-.. tip::
+.. note::
 
-   Want to have a group conversation with more than 7 people? You can :doc:`create a private channel </collaborate/create-channels>`. Alternatively, from Mattermost v9.1, you can :doc:`convert the group message to a private channel </collaborate/convert-group-messages>`.
+   - Want to have a group conversation with more than 7 people? You can :doc:`create a private channel </collaborate/create-channels>`. Alternatively, from Mattermost v9.1, you can :doc:`convert the group message to a private channel </collaborate/convert-group-messages>`.
+  
+   - Any group message history you have with a deactived user remains available :ref:`unless your systems administrator disables your ability to do so <configure/site-configuration-settings:allow users to view archived channels>`.
```
Comment (cwarnermm) [inline source/collaborate/channel-types.rst:49]:
```suggestion
   - Any group message history you have with a deactivated user remains available :ref:`unless your system admin disables your ability to do so <configure/site-configuration-settings:allow users to view archived channels>`.
```

**Review by cwarnermm — APPROVED** [review]
Thanks, @TomerPacific!

**Comment by Rutam21** [comment]
@lindy65 Please assign this PR to me for QA Testing. Thanks.

**Comment by lindy65** [comment]
Thanks, assigned to you @Rutam21

**Comment by Rutam21** [comment]
### The changes in this PR have been tested in the preview environment and all the required changes are available in the mentioned pages.

- The `Direct Messages` and `Group Messages` `Note` sections contain a new bullet point with the required information.
- The link in the text `unless your system admin disables your ability to do so` for that specific bullet point is working fine and points to the correct place in the Docs.

I've attached a video as a confirmation.

https://github.com/user-attachments/assets/339b3a00-37d3-464b-821f-c710ae07fa6d

cc: @lindy65

**Comment by lindy65** [comment]
Thank you @Rutam21 - counted towards Hacktoberfest!

cc @cwarnermm

**Pull request merged**